### PR TITLE
Role descriptions

### PR DESCRIPTION
**Pull request by lbartoletti**
The PR adds a description of each role

### CODE REVIEW

**Comment by lbartoletti** [comment]
Same for QGEP https://github.com/QGEP/docs/pull/66

File: installation-guide/database-initialization.rst [hunk 1]
```diff
@@ -63,6 +63,13 @@ The script has the following options:
 - ``-d`` or ``--drop-schema``    drop schemas (cascaded) if they exist
 - ``-r`` or ``--create-roles`` create roles in the database
 
+Roles are: ``qgep_viewer``, ``qgep_user``, ``qgep_manager``, ``qgep_sysadmin``.
```
Comment (ponceta) [inline installation-guide/database-initialization.rst]: qwat_viewer, qwat_user, qwat_manager, qwat_sysadmin

**Comment by vpicavet** [comment]
Typo "qgep" -> "qwat"

**Comment by ponceta** [comment]
Thanks! Can be merged!

**Pull request merged**